contributing-guides/style-guide.md

```diff
@@ -96,9 +118,9 @@ Use backticks on the following:
 
 ## Imperative Mood
 
-Example descriptions have to be phrased in imperative mood.  
-For example, use `List all files`, instead of `Listing all files` or `File listing`.  
-This also applies to all translations by default, unless this is not possible for some reason.  
+Example descriptions have to be phrased in imperative mood.
```

Comment:
The double spaces at the end of each line here force markdown renders to put text on the next line - just like `<br />` in HTML. Let's keep this as-is. Same goes for all other instances below.

Reply:
It's due to the `.editorconfig` setting:

```
trim_trailing_whitespace = true
```